app/templates/courses/activate-licenses-modal.jade

```diff
@@ -51,7 +52,22 @@ block modal-body-content
           label
             input.user-checkbox(type="checkbox", disabled=true, checked=true, data-user-id=user.id, name='user')
             span.spr= user.broadName()
-    
+  form.form.m-t-3
+    .prepaid-type-line
+      span(data-i18n="teacher.select_license_type")
+        span :
+      span(data-i18n="inventory.available_item")
+    #selectPrepaidType.well.form-group
+      - var prepaids = view.prepaidByGroup
+      - console.log(prepaids)
```

Comment:
Let's remove logging we don't need; if it's really useful going forward, then keep it in but add a label or description to the log line.